README.md

```diff
@@ -1,7 +1,41 @@
-Android Templates
-===================================
+<p align="center">
+  <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png" />
+</p>
+
+---
 
 A collection of templates:
 
 * **[CoroutineTemplate](https://github.com/nimblehq/android-templates/tree/kotlin/CoroutineTemplate)**
 * **[RxJavaTemplate](https://github.com/nimblehq/android-templates/tree/kotlin/RxJavaTemplate)**
+
+## Setup
+
+1. Clone or Download this repository then extract and open the folder
+2. Run `newproject.sh` script to create a copy with the inputs:
```

Comment:
Small wording improvements:
```suggestion
1. Clone or download this repository to your local machine, then extract and open the folder
2. Run `newproject.sh` script to create a new project copy with the following inputs:
```

Reply:
`create a new project` is ok, `copy` == no changes, but we will change the templated inputs